screeps-game-api/CHANGELOG.md

```diff
@@ -1,14 +1,27 @@
 Unreleased
 ==========
 
+
+0.4.0 (2019-02-??)
```

Comment:
pick a date

Reply:
I really should but I figured the changelog date doesn't matter as much as just merging it as soon as it can be.